developer_documentation/contribution_guidelines.Rmd

```diff
@@ -28,15 +26,14 @@ Developer's Manual, and find the chapter on contributing to BioCro.
 
 [Solar Position module]: `r params$biocro_root`/src/module_library/solar_position_michalsky.h "Source code for the Solar Position module" {target="_blank"}
 
-<!-- CHANGE THIS URL WHEN MOVING FROM ebimodeling/biocro TO ebimodeling/biocro!!! -->
-[Solar Position module rendering]: https://ebimodeling.github.io/biocro-documentation/master/doxygen/doxygen_docs_modules_public_members_only/classstandard_b_m_l_1_1solar__position__michalsky.html#details "Documentation of the Solar Position module" {target="_blank"}
+[Solar Position module rendering]: https://biocro.github.io/BioCro-documentation/latest/doxygen/doxygen_docs_modules_public_members_only/classstandard_b_m_l_1_1solar__position__michalsky.html#details "Documentation of the Solar Position module" {target="_blank"}
 
-[dimensions]: https://en.wikipedia.org/wiki/International_System_of_Quantities#Base_quantities "Quantities and their dimensions (Wikipedia)" {target="_blank"}
+[dimensions]: https://en.wikipedia.org/wiki/International_System_of_Quantities#Dimensional_expression_of_derived_quantities "Dimensional expression of derived quantities (Wikipedia)" {target="_blank"}
```

Comment:
This new link doesn't list the dimensions for units.

Reply:
I noticed this too, but if you check to see where the link is referenced, it's in the context of using dimensions rather than units, so the new link might be more appropriate. I think there are other references to coherent SI units with different links.